runtime_tests/test_samv71_runtime.py

```diff
@@ -124,3 +124,146 @@ def test_samv71_ada_og(taste_project):
     # Check expected compilation output
     stderr = build.stderr.decode('utf-8')
     assert build.returncode == 0, 'Compilation errors: \n{}'.format(stderr)
+
+##
+# TASTE SAMV71 runtime supports deployment of TASTE models containing at least 8 TASTE functions (including
+# at least 4 EDS based). This is validated by the demonstration application (DemoSat2) which contains 14 EDS functions
```

Comment:
8 EDS functions (see ADD). The rest are C-based, 13 in total.

Reply:
changed